Dockerfile

```diff
@@ -8,10 +8,11 @@ ARG CPANMOPTS=
 ######################
 # Base modperl image stage
 ######################
-FROM debian:bullseye AS modperl
+FROM debian:bookworm AS modperl
```

Comment:
@alexgarel I noticed that you moved to `bullseye` in #8208 just a few weeks ago. Unfortunately, the `libzxing-dev` version [required by `Imager::zxing`](https://github.com/tonycoz/imager-zxing/blob/fc9ff87e2edf35df3317cf8467f110b17e124910/README#L5-L6) is only available in `bookworm`. If the intention is to stay on `bullseye`, especially if there's still some bare-metal install remaining (is it?), we may have to see how we can achieve that.

Reply:
After some recent work on Imager::zxing I'd recommend installing zxing-cpp 2.1.0 from source rather than using the bookworm packaged version of the library.

I've had some images (see tonycoz/imager-zxing#2) supplied by another user that causes an assertion failure when trying to parse them, both with git Imager::zxing and with the Debian built ZXingReader tool that's part of the zxing-cpp-tools package.

This assertion was previously reported upstream as zxing-cpp/issues/399 and is apparently fixed in zxing-cpp 2.0.0, though I've only tested 2.1.0.

Reply:
Thanks so much for the heads-up, @tonycoz! I prepared using zxing-cpp 2.1.0, but since `debian:bullseye` uses an old version of GCC/G++, C++17 needs to be enabled explicitly. I'll just wait until https://github.com/tonycoz/imager-zxing/commit/829a1293ce8836f46bc93b55feac909abf07737e hits CPAN. 😄

Reply:
Done 😄